### ISSUE TITLE
Since 0.116 Meross devices became not available 

### ISSUE THREAD

**Issue by liuk4friends** [albertogeniola/meross-homeassistant#149]
## The problem

As described in the title, the 0.116 update has effectively blocked the functioning of this custom component.

## Environment

- Home Assistant Core release with the issue:  0.116
- Last working Home Assistant Core release (if known): 0.115 
- Operating environment (OS/Container/Supervised/Core): HassOS

```yaml

```

## Traceback/Error logs

```txt
Logger: meross_iot.manager
Source: /usr/local/lib/python3.8/site-packages/meross_iot/manager.py:538 
First occurred: 23:24:03 (2 occurrences) 
Last logged: 23:24:03

Timeout occurred while waiting a response for message b'{"header": {"from": "/app/80300-1df26c8cd1cdbe6cb7db4e8a2e2bcd79/subscribe", "messageId": "29d42c5abc91e99bd333925c253d1719", "method": "GET", "namespace": "Appliance.System.Ability", "payloadVersion": 1, "sign": "bf8b60b5b816800e862080397fd3877d", "timestamp": 1602105839}, "payload": {}}' sent to device uuid 18092713565378251a0834298f16a85a. Timeout was: 5.0 seconds
Timeout occurred while waiting a response for message b'{"header": {"from": "/app/80300-1df26c8cd1cdbe6cb7db4e8a2e2bcd79/subscribe", "messageId": "e1724c770c0296ec585a653dab5b3afa", "method": "GET", "namespace": "Appliance.System.Ability", "payloadVersion": 1, "sign": "9f3d12a70faf105f56c88a0bd0388375", "timestamp": 1602105839}, "payload": {}}' sent to device uuid 1808175274038329083134298f161caa. Timeout was: 5.0 seconds


Logger: meross_iot.manager
Source: /usr/local/lib/python3.8/site-packages/meross_iot/manager.py:285 
First occurred: 23:24:03 (2 occurrences) 
Last logged: 23:24:03

Failed to retrieve abilities for device Piantana (18092713565378251a0834298f16a85a). This device won't be enrolled.
Failed to retrieve abilities for device Ciabatta (1808175274038329083134298f161caa). This device won't be enrolled.
```

## Additional information

No additional information

**Comment by Keltere** [comment]
On `custom_components/meross_cloud/manifest.json `
replace the 0.4.0.1 version with 0.4.0.5
`"requirements": ["meross-iot==0.4.0.1"],`
to
`"requirements": ["meross-iot==0.4.0.5"],`
then restart home assistant from the control panel

using visual studio code it will be really easy

**Comment by maxcanna** [comment]
I confirm this fix as working

**Comment by uvjim** [comment]
This worked for me too.

**Comment by cconnoruk** [comment]
Snap, thank you :).

**Comment by silviudc** [comment]
Same here, thanks 👍

**Comment by PabloTheNinja** [comment]
I initially rolled back to 0.115 and V1 and the devices were available again - implemented this fix and updated to 0.116 and all devices now available.

The change to "requirements": ["meross-iot==0.4.0.5"] I did through file editor.

Many thanks.

**Comment by ser1993** [comment]
It worked for me also. Will this be implemented in the next update? 

Many thanks.

**Comment by Keltere** [comment]
> It worked for me also. Will this be implemented in the next update?
> 
> Many thanks.

yas, got from there

**Comment by Florian-Schmidt** [comment]
Unbelievable, works!

**Comment by remco1985** [comment]
Thank you for the solution!

**Comment by liuk4friends** [comment]
Thank you. It worked for me too.
I wonder if I should close this issue...

**Comment by moto3001** [comment]
Wow this fix actually resolved also _another_ meross warning I was previously getting.  Thanks!

**Comment by albertogeniola** [comment]
Thank you guys for letting me know. I've just released [v1.0.3](https://github.com/albertogeniola/meross-homeassistant/releases/tag/v1.0.3) which targets meross_iot 0.4.0.5. 
Great work @Keltere .
